component/org.wso2.carbon.identity.api.otp.service.emailotp/pom.xml

```diff
@@ -0,0 +1,190 @@
+            <artifactId>org.wso2.carbon.identity.application.authentication.framework</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.extension.identity.authenticator.outbound.emailotp</groupId>
+            <artifactId>org.wso2.carbon.extension.identity.emailotp.common</artifactId>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <finalName>org.wso2.carbon.identity.api.otp.service.emailotp</finalName>
+        <plugins>
+<!--                    <plugin>-->
+<!--                        <groupId>org.openapitools</groupId>-->
+<!--                        <artifactId>openapi-generator-maven-plugin</artifactId>-->
+<!--                        <executions>-->
+<!--                            <execution>-->
+<!--                                <goals>-->
+<!--                                    <goal>generate</goal>-->
+<!--                                </goals>-->
+<!--                                <configuration>-->
+<!--                                    <inputSpec>${project.basedir}/src/main/resources/email-otp.yaml</inputSpec>-->
+<!--                                    <generatorName>org.wso2.carbon.codegen.CxfWso2Generator</generatorName>-->
+<!--                                    <configOptions>-->
+<!--                                        <sourceFolder>src/gen/java</sourceFolder>-->
+<!--                                        <apiPackage>org.wso2.carbon.identity.api.otp.service.emailotp</apiPackage>-->
+<!--                                        <modelPackage>org.wso2.carbon.identity.api.otp.service.emailotp.dto</modelPackage>-->
+<!--                                        <packageName>org.wso2.carbon.identity.api.otp.service.emailotp</packageName>-->
+<!--                                        <dateLibrary>java8</dateLibrary>-->
+<!--                                        <hideGenerationTimestamp>true</hideGenerationTimestamp>-->
+<!--                                        <ignoreFileOverride>${project.basedir}/.openapi-generator-ignore</ignoreFileOverride>-->
+<!--                                    </configOptions>-->
+<!--                                    <output>.</output>-->
+<!--                                    <skipOverwrite>false</skipOverwrite>-->
+<!--                                </configuration>-->
+<!--                            </execution>-->
+<!--                         </executions>-->
+<!--                         <dependencies>-->
+<!--                            <dependency>-->
+<!--                                <groupId>org.openapitools</groupId>-->
+<!--                                <artifactId>cxf-wso2-openapi-generator</artifactId>-->
+<!--                                <version>1.0.0</version>-->
+<!--                            </dependency>-->
+<!--                         </dependencies>-->
+<!--                    </plugin>-->
```

Comment:
Can't we remove these commented lines?

Reply:
These are the configurations of the `open-api` tool, so basically, we need this if we update the `YAML file.